### PR TITLE
fix(repo-doctor): Send mattermost notif to front

### PR DESCRIPTION
**Pull request by trollepierre**
Adding Message on Mattermost with result of Mattermost: 
https://mattermost.cozycloud.cc/cozycloud/pl/pxpswtqtfpgt8cambgc4qyduwa

### CODE REVIEW

**Comment by trollepierre** [comment]
I set it as draft, until I create the Travis job on the master pipeline.

File: .travis.yml [hunk 1]
```diff
@@ -23,12 +20,21 @@ deploy:
     on:
       branch: master
       repo: cozy/cozy-libs
+
+jobs:
+  include:
+    - name: 'Repo Doctor'
+      script: if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then cd ./packages/repo-doctor && yarn start:mattermost; fi
```
Comment (sblaisot) [inline .travis.yml:27]: nit: 

```patch
- script: if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then cd ./packages/repo-doctor && yarn start:mattermost; fi
+ script: '[ "$TRAVIS_PULL_REQUEST" != "false" ] && pushd ./packages/repo-doctor && yarn start:mattermost && popd'
```

**Review by sblaisot — APPROVED** [review]
one small nit, otherwise LGTM

File: .travis.yml [hunk 2]
```diff
@@ -19,16 +16,25 @@ before_deploy:
...
-    script: git checkout master && git remote set-url origin https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-libs.git && echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc && yarn lerna publish --yes -m "[skip ci] Publish"
+    script: 'git checkout master && git remote set-url origin https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-libs.git && echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc && yarn lerna publish --yes -m "[skip ci] Publish"'
     on:
       branch: master
       repo: cozy/cozy-libs
+
+jobs:
+  include:
+    - name: 'Repo Doctor'
+      script: '[ "$TRAVIS_PULL_REQUEST" != "false" ] && pushd ./packages/repo-doctor && yarn start:mattermost && popd'
```
Comment (trollepierre) [inline .travis.yml:27]: en fait, mieux vaut lancer le job uniquement sur master ;-)

Comment (trollepierre) [reply .travis.yml]: actually let's trigger an automatic cron sending this report once a week

**Review by acezard — APPROVED** [review]

**Pull request merged**